subsys/usb/device/Kconfig

```diff
@@ -48,6 +48,21 @@ config USB_DEVICE_SN
 	  Serial Number String will be derived from
 	  Hardware Information Driver (HWINFO).
 
+config USB_DEVICE_CONFIGURATION_ENABLE
+	bool "USB configuration descriptor support"
+	default n
+	help
+	  Select 'y' to make the USB configuration descriptor available. The
+	  configuration descriptor is not available by default.
+
+if USB_DEVICE_CONFIGURATION_ENABLE
+config USB_DEVICE_CONFIGURATION
```

Comment:
nit: add a blank line before this and one before the endif below

Reply:
Done. Thanks.